### PR TITLE
refactor index

### PR DESCRIPTION
**Pull request by freeman-lab**
Reorganizes our `research` contents browser to support the following features requested by the team:
- multiplexed article types (an single entry can refer both to an article and a tool on the same topic)
- filtering by tags (as a complement to the multiplexing)
- arbitrary links (so an entry can point entirely to an external resource)

### CODE REVIEW

File: components/entry.js [hunk 1]
```diff
@@ -109,56 +66,112 @@ const Entry = ({ info }) => {
...
+    return (
+      <NextLink href={url} passHref={true}>
+        <Link sx={{ textDecoration: 'none' }}>{children}</Link>
+      </NextLink>
+    )
+  } else {
+    return (
+      <Link sx={{ textDecoration: 'none' }} href={url}>
+        {children}
+      </Link>
```
Comment (jhamman) [inline components/entry.js:172]: For tracking, I have a slight preference to using a our [`TaggedLink`](https://github.com/carbonplan/research/blob/e7d4cae1dad54aad6958052563cac02e067d4294/components/tagged-link.js#L11) component here:

```js
<TaggedLink action={action} category={type} href={url}>{children}</TaggedLink>
```

where `category` is either `downloads` or `external`

and `action` is either `PDF` or `tool` or `website`

File: articles/permanence-calculator-explainer/index.md [hunk 2]
```diff
@@ -311,9 +315,17 @@ Our calculator puts projects that temporarily remove CO₂ from the atmosphere o
...
+
+Please cite as:
+
+D Cullenward, J Hamman, J Freeman (2020) “The cost of temporary carbon removal” CarbonPlan https://carbonplan.org/research/permanence-calculator-explainer
+
+## Terms
 
-CarbonPlan received a grant from the ClimateWorks Foundation to support this work. ClimateWorks did not exercise any control over the output. CarbonPlan thanks Gernot Wagner, Frances Moore, Toly Rinberg, and Rafael Broze for helpful feedback. CarbonPlan is solely responsible for the content of this writeup, which does not necessarily reflect the views of ClimateWorks, those who provided feedback, or any other individuals or organizations.
+CarbonPlan received a grant from the ClimateWorks Foundation to support this work. ClimateWorks did not exercise any control over the output. CarbonPlan is solely responsible for the content of this writeup, which does not necessarily reflect the views of ClimateWorks, those who provided feedback, or any other individuals or organizations.
```
Comment (jhamman) [inline articles/permanence-calculator-explainer/index.md:328]: I wonder if we could add some design around these sections. Just feels a bit bland.

File: articles/soil-carbon-comment/index.md [hunk 3]
```diff
@@ -172,7 +176,11 @@ But carbon credits aren’t the only way to support soil carbon. An alternative
 
 Meanwhile, any efforts built around carbon offset credits should be transparent, subject to third-party verification, and capable of thorough validation by financially independent organizations.
 
-## Terms of engagement
+## Credits
```
Comment (jhamman) [inline articles/soil-carbon-comment/index.md:179]: I suggest we bump the version number to 1.0.1 for the article and add a section above that reads something like:

```markdown
## Update 8/25/2020

After reviewing an updated version of the draft protocol, we submitted a second <TaggedLink action='PDF' category='downloads' href='https://carbonplan-assets.s3.amazonaws.com/docs/Soil-Carbon-Comment-Letter-08-25-2020.pdf'>comment letter</TaggedLink> raising additional issues with the proposed protocol.
```

File: articles/permanence-calculator-explainer/index.md (hunk 2 above)
Comment (freeman-lab) [reply articles/permanence-calculator-explainer/index.md:328]: went with something simple for now (divider lines), we can improve further in the future

**Pull request merged**